app/src/main/java/org/mozilla/fenix/addons/InstalledAddonDetailsFragment.kt

```diff
@@ -149,6 +151,7 @@ class InstalledAddonDetailsFragment : Fragment() {
                         runIfFragmentIsAttached {
                             switch.isClickable = true
                             view.remove_add_on.isEnabled = true
+                            switch.setState(currState)
```

Comment:
To be safe should we fallback to `switch.setState(addon.isEnabled())` instead? As it will hold the latest successful `isEnabled` value, because `currState` capture the value of `addon.isEnabled()` when `bindEnableSwitch` was executed, and it could contain the wrong value as multiple successful to `enableAddon/disableAddon `can happen in between

Reply:
The same applies to `bindAllowInPrivateBrowsingSwitch`.
Another alternative is to move `val currState = addon.isEnabled()` inside of `switch.setOnCheckedChangeListener { v, isChecked ->`

Reply:
Sounds good to me.